service/src/test/java/bio/terra/cbas/initialization/TestCloneRecoveryService.java

```diff
@@ -51,9 +49,8 @@ void generateTemplateUpdateManifest() {
     // all other clone run sets should be listed as non-templates.
     assertEquals(1, manifest.keepAsTemplate().size());
     assertEquals(2, manifest.toBeDeleted().size());
```

Comment:
Minor - now that you are testing exact membership, I think these counts are automatically validated too

Reply:
ah, yes you're right